css/style.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background: #eceef1;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header a { padding-bottom: 5px; }
+
+footer a { padding: 5px; }
+
+a:hover,
+a:visited,
+a:link,
+a:active {
+  color: #2478c5;
+  text-decoration: none;
+}
+
+footer a:hover { color: #000; }
+
+footer a:visited { color: #000; }
+
+footer a:link { color: #000; }
+
+footer a:active { color: #000; }
+
```

Comment:
Don`t repeat your code you can combine elements and  write instead of your code from line 18 till 25 next code

footer a:hover,
footer a:visited,
footer a:link,
footer a:active { 
  color: #000; 
}

Read more about here [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)

![screen](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png)